action/wellknown.go

```diff
@@ -1,55 +1,9 @@
 package action
 
-import (
-	"github.com/cnabio/cnab-go/claim"
-	"github.com/cnabio/cnab-go/credentials"
-	"github.com/cnabio/cnab-go/driver"
-)
-
 // Well known constants define the Well Known CNAB actions to be taken
 const (
```

Comment:
Will the well known action list expand over time?
Does it make sense to keep this in a separate file?

Reply:
Ah, probably not. I think we are done identifying well-known actions in the spec. Now that this file is just this const, I can move the definitions to action.go.